nixos/modules/services/monitoring/riemann-tools.nix

```diff
@@ -34,8 +37,24 @@ in {
           Address of the host riemann node. Defaults to localhost.
         '';
       };
+      attributes = mkOption {
+        type = types.listOf types.string;
+        default = [];
+        description = ''
+          A list of attributes and their values (name=value) added to the events.
+        '';
+        example = ["myattribute=42"];
+      };
+      commandSwitches = mkOption {
```

Comment:
There's some precedence for calling this `extraArgs`, so I think we should stick with that.